main.go

```diff
@@ -96,12 +86,6 @@ func main() {
 	if cfg.Api.Enabled {
 		go startApi()
 	}
-	if cfg.BlockUnlocker.Enabled {
-		go startBlockUnlocker()
-	}
-	if cfg.Payouts.Enabled {
```

Comment:
In this case you have removed the payout processing. Maybe separate it out if you want to look into it in the future.

Reply:
https://github.com/dominant-strategies/go-quai-stratum/pull/8#discussion_r1115931194

Reply:
https://github.com/dominant-strategies/go-quai-stratum/issues/8
https://github.com/dominant-strategies/go-quai-stratum/issues/9